package.json

```diff
@@ -44,7 +47,7 @@
     "element-class": "^0.2.0"
   },
   "peerDependencies": {
-    "react": ">=0.13.3"
+    "react": ">=0.14.0.rc1"
```

Comment:
@rogeliog Thanks for your great work. I'm currently using your fork now but getting an error on `npm install` because of this. I guess it should be `-rc1` instead of `.rc1`.